src/main/java/textcolors/TextColor.java

```diff
@@ -0,0 +1,19 @@
+package textcolors;
+
+/**
+ * This class contains different ANSI color codes which can be used to color texts.
+ */
+public class TextColor {
+    /**
+     * ANSI codes for different colors.
+     */
+    public static final String COLOR_RESET = "\u001B[0m";
+    public static final String COLOR_BLACK = "\u001B[30m";
+    public static final String COLOR_RED = "\u001B[31m";
+    public static final String COLOR_GREEN = "\u001B[32m";
+    public static final String COLOR_YELLOW = "\u001B[33m";
+    public static final String COLOR_BLUE = "\u001B[34m";
+    public static final String COLOR_PURPLE = "\u001B[35m";
+    public static final String COLOR_CYAN = "\u001B[36m";
+    public static final String COLOR_WHITE = "\u001B[37m";
```

Comment:
Didn't know you could do that. Very cool!

Reply:
Thank you!